src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -227,8 +227,8 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({
     }
   }
 
-  // true when scale larger than initial zoom
-  const isZoomedIn = scale > 1.07
+  // true when scale larger than "safe" zoom, due to viewport
+  const isZoomedIn = scale > 1.5
```

Comment:
It's fine as hotfix, but looking at evolution here from `parseFloat(scale.toFixed(2)) > 1` to `scale > 1.005` to `scale > 1.07` to `scale > 1.5`. Why are setting zoom flag in render cycle. Maybe putting it in something like `useLayoutEffect` will fix that (no idea, just throwing ideas).

Reply:
Ok I think it stopped working, because we changed styles lately in `QuickPinchZoomOuter > div`. We apply styles to that third party container and it sets initial zoom factor based on container and child dimensions. Ideally we should not apply custom styled `QuickPinchZoom`, but whatever 

If above is true
1. Can you add a comment to `QuickPinchZoomOuter` that any changes affect `isZoomedIn` scale factor?
2. We could limit scale issue by moving most styles to outer component (limit `  '> div'`)
```
const QuickPinchZoomOuter = styled('div')(({ theme }) => ({
  position: 'absolute',
  inset: 0,
  borderWidth: 10,
  borderStyle: 'solid',
  borderRadius: '50%',
  borderColor: theme.palette.layout.darkBorder,
  '> div': {
    position: 'absolute',
    inset: 0,
  },
  [theme.breakpoints.up('sm')]: {
    borderWidth: 15,
  },
}))
```
3) Don't apply '> div'` styles at all, but no idea if that's possible

Reply:
Right, moved into bellow useEffect, which sets zoomedIn the parent component.

Reply:
Right, this is required to the border, in order case it would beed to be included into svg. Which makes it more difficult to display the border on top of the graph.